### DISCUSSION TITLE
Model Upload Suggestions

### DISCUSSION THREAD

**Discussion by duskfallcrew** [civitai/civitai#429] [Ideas]
Requirements should be:
Model has description of a tleast 300 chars 
Model has at LEAST 3 images.

Preferably "Discussion off" is only left for moderation team so many models are getting randomly uploaded with no information

**Comment by duskfallcrew** [comment]
(also yes peeps this is duskfall i dont have a renamed github account XD)

**Comment by TroubleDarkness** [comment]
A good idea, i think. Sometimes it really a shame, that someone may just not tell how to use models or LoRAs. Or maybe authors just don't tell the main advantages or disadvantages of mixes/models, they are not telling the recipes (which is unecessary, but, well, a good addition to model/mix/LoRA description), and finally, sometimes they are not telling anything at all, just the blank page with download button. But i'm sure CivitAI team won't even read this, because they have more important things to do - make site worse and user-friedly no more.

For me the real promblem is chinese/japanese langauage in model/mixes/loras names (mostly) and descriptions, which is hard to translate properly, and i've heard of this problem many times.

**Comment by wavjav** [comment]
The worse thing about Civitai is they change their system/Ui/UX waaaaaaaaaaaaaaaaay to often....
When I finally adapt to their Ugly UI, they change it 2 days later, when I try to adapt with their new ugliness and feel comfort even though their ugly they change it again 2 days later with more wooooooooooooooorse user experience smh

**Reply by duskfallcrew** [reply]
OK but that's not entirely relative to the discussion, Ui/UX does matter yes - and it CAN get frustrating btu they're TRYING to fix usability, not crap on your style. XD  It's a VERY difficult thing to do when a website has to adapt on the fly.
